cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -40,14 +40,14 @@ To use the **Docker image**:
 
 - Once Docker has started, pull down the *CDAP Docker Image* from the Docker hub using::
 
-    $ docker pull caskdata/cdap-standalone
+    $ docker pull caskdata/cdap-standalone:\ |literal-release|
```

Comment:
It does not work.
![screen shot 2015-06-23 at 2 52 51 pm](https://cloud.githubusercontent.com/assets/380021/8318193/9736c96a-19b7-11e5-96cd-8bd67c153979.png)